biztalk/core/how-to-modify-group-settings.md

```diff
@@ -38,10 +38,19 @@ Using the Settings Dashboard, you can modify the configuration information used
+   |              **Retry interval**                       | New starting with BizTalk Server 2020.<br /><br /> Set the interval in which BizTalk server attempts to recover receive location from failures.                                                                        | 1 – 43200 |       60       |              -               |                                                  -                                                      |
    |              **Tracking and Reporting**               |                                                                                                                                                                                                                                                                                                                                                                                               |                 -                 |       -       |                                                      -                                                      |
    | **Message box performance counter sampling interval** |                                                                       Set the interval at which performance counters are refreshed.<br /><br /> The interval trades off load on database versus up-to-dateness of counters. The higher value means less frequently updated data, and thus lesser load on the database.                                                                        | 1 – Maximum value of type Integer |       -       |               Largest value on any machine in the BizTalk group if present. If not, default.                |
    |            **Enable group level tracking**            | Select this option to turn the group level tracking for BizTalk Server on.<br /><br /> Turning off global tracking disables the tracking interceptors for the entire [!INCLUDE[btsBizTalkServerNoVersion](../includes/btsbiztalkservernoversion-md.md)] group. This means, BizTalk Server will not track events in its tracking tables. **Note:**  This setting does not impact BAM Tracking. |              On, Off              |      On       |                                                      -                                                      |
-
+   |            **Allow import of tracking settings**      | New starting with BizTalk Server 2026.<br /><br /> Select this option to turn the group level tracking for BizTalk Server on.<br /><br /> Turning off global tracking disables the tracking interceptors for the entire [!INCLUDE[btsBizTalkServerNoVersion](../includes/btsbiztalkservernoversion-md.md)] group. This means, BizTalk Server will not track events in its tracking tables. **Note:**  This setting does not impact BAM Tracking. |              On, Off              |      On       |
```

Comment:
this description seems same as 'Enable group level tracking' - copy paste error?

Reply:
fixed